lms/static/sass/views/_financial-assistance.scss

```diff
@@ -120,6 +120,7 @@
     border-bottom: 2px solid $gray-l5;
     padding: 20px 0;
     margin-bottom: 20px;
+    width: 150%;
```

Comment:
Instead of doing this, I believe we should increase `max-width: 500px` applying on `.financial-assistance-form`.

Reply:
Well the max-width is already set to 500 px. Increasing max-width you are able to fix the issue but also causes the whole form to get expanded, which in my opinion doesn't look good and gives an impression of something new and its good to be as specific as possible without giving an impression of UI change. My change increase the size of that specific div while keeping the same form factor intact.
 
**Original form with max-width: 500px**
![Screenshot from 2019-11-18 10-37-40](https://user-images.githubusercontent.com/373677/69027361-810aef80-09f0-11ea-9953-b6962864150f.png)

**With form expansion (.financial-assistance-form)**
![Screenshot from 2019-11-18 10-35-28](https://user-images.githubusercontent.com/373677/69027389-97b14680-09f0-11ea-9319-13de843049a5.png)

**With specific div expansion (user-info)**
![Screenshot from 2019-11-18 10-33-27](https://user-images.githubusercontent.com/373677/69027447-c3ccc780-09f0-11ea-9904-6846fe361ffb.png)

Reply:
Increasing the max width will align the user info row with the whole form but increasing width of a specific row is increasing the width to right side of the form and looks little odd to me, isn't it?
![screencapture-courses-edx-org-financial-assistance-apply-2019-11-18-12_43_21](https://user-images.githubusercontent.com/2851134/69033836-aeac6480-0a01-11ea-934b-1f10fa277b7b.png)

### After increasing form's max width:
![screencapture-courses-edx-org-financial-assistance-apply-2019-11-18-12_44_28](https://user-images.githubusercontent.com/2851134/69033948-eddab580-0a01-11ea-8d83-ed74a1d54e70.png)

Reply:
Fixed according to your suggestion.